app/src/main/java/com/android/shelfLife/ui/overview/EditFoodItem.kt

```diff
@@ -146,13 +146,14 @@ fun EditFoodItemScreen(
                               textAlign = TextAlign.Start)
                         }
                       }
-                      Spacer(modifier = Modifier.width(8.dp))
-                      // Unit
-                      // TODO this doesnt look good on the UI
+
                       Card(
                           border = CardDefaults.outlinedCardBorder(),
+                          shape = MaterialTheme.shapes.large,
```

Comment:
Good job implementing the nice UI changes! I see you correctly built upon my feedback from #162